### DISCUSSION TITLE
Pro badge is not showing

### DISCUSSION THREAD

**Discussion by dev-aniketj** [community/community#45056] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I subscribe the GitHub education pack!!!
But **Pro** badge is not showing!

**Comment by jge162** [comment, marked as answer]
Has it been 24 hours? Also did u get an email saying you were approved? It will show up. Mine took 24 hours. @dev-aniketj

**Reply by jge162** [reply]
Also go back to the website and login. It will tell u of approved or not. https://education.github.com/benefits

**Reply by dev-aniketj** [reply]
Yes, approved 2 days ago,
In **edit profile** is show enabled check box for **Pro**.
I already enabled it but nothing happened.

**Reply by jge162** [reply]
After you login it will say approved. If not do the submission again.

**Reply by dev-aniketj** [reply]
Approved Submitted Jan 20, 2023
Congrats! Your request for [@dev-aniketj](https://github.com/dev-aniketj) was approved.

**Reply by jge162** [reply]
> Yes, approved 2 days ago,
> In **edit profile** is show enabled check box for **Pro**.
> I already enabled it but nothing happened.

Oh that’s a support issue then. Reach out to them. They can help. https://support.github.com/

**Reply by jge162** [reply]
Type pro badge. In search bar

**Reply by dev-aniketj** [reply]
It didn’t find out the answer, please help me.

**Reply by dev-aniketj** [reply]
@courtneycl @github-staff  read this topic, help me

**Reply by jge162** [reply]
Check your profile settings: Go to your GitHub profile settings and make sure that your GitHub Pro badge is enabled.
Clear your browser cache: Clearing your browser cache can sometimes help resolve issues with badges not showing up.

**Reply by jge162** [reply]
Go to https://github.com/contact

**Reply by jge162** [reply]
Hope it shows up for u

**Reply by onwp** [reply]
Mine didn't show up yet. I will check after 24 hrs again.

**Comment by radar-blip** [comment]
![proooo](https://user-images.githubusercontent.com/39196767/213937139-9bbb4741-6595-4308-ae50-a5193ac4e558.png)

**Reply by jge162** [reply]
Finally showed up.

**Reply by radar-blip** [reply]
Yup, just took some time.

**Reply by jge162** [reply]
Nice

**Reply by dev-aniketj** [reply]
Thanks @jge162 dude!!!
